src/main/java/org/gitlab4j/api/models/MembershipSourceType.java

```diff
@@ -0,0 +1,8 @@
+package org.gitlab4j.api.models;
+
+public enum MembershipSourceType {
```

Comment:
In this project all enum names should be capitalized. Can you modify this enum like [PackageType](https://github.com/gitlab4j/gitlab4j-api/blob/master/src/main/java/org/gitlab4j/api/models/PackageType.java)

Reply:
Oh, sorry. Didn’t notice this coding convention.
I’ve adjusted the PR as you suggested.

Reply:
Thanks